lib/channel.js

```diff
@@ -132,6 +137,36 @@ class Channel {
     this.error(500, new Error('Packet structure error'));
   }
 
+  stream(data) {
+    const chunkStart = data.indexOf('}') + 1;
```

Comment:
There will be an issue when a metadata will have the nested json objects. Using the `lastIndexOf` method could prevent that.